README.md

```diff
@@ -237,3 +236,7 @@ additional required environment variables encrypted with
 [sops](https://github.com/mozilla/sops):
 
     sops --decrypt --in-place --input-type dotenv --output-type dotenv --gcp-kms projects/radicle-services/locations/global/keyRings/sops/cryptoKeys/sops-key .env
+
```

Comment:
Could we maybe introduce a separate section here named something along the lines of "(git) files ownership" to make it abundantly clear what's the purpose of that nuisance that the additional env var is?

Reply:
Yeah sounds good